src/main/java/lotto/constant/LottoRank.java

```diff
@@ -0,0 +1,34 @@
+package lotto.constant;
+
+import java.util.Arrays;
+
+public enum LottoRank {
+    NONE(0, 0),
+    FIFTH(3, 5000),
+    FOURTH(4, 5_0000),
+    THIRD(5, 150_0000),
+    FIRST(6, 20_0000_0000);
+
+    int matchingCount;
+    int price;
```

Comment:
```suggestion
    private final int matchingCount;
    private final int price;
```

접근제어자를 생략하신 이유가 있을까요?

Reply:
아니요 이건 못보고 실수했습니다.. 😭
수정하겠습니다